.github/workflows/ci.yml

```diff
@@ -434,6 +434,8 @@ jobs:
       run:
         shell: bash -l {0}
     env:
+      MODIN_EXPERIMENTAL: "True"
+      MODIN_ENGINE: "native"
```

Comment:
We allowed this bug to sneak into the master because the PR's CI was configured differently compared with the master's CI (check [this comment](https://github.com/modin-project/modin/issues/5536#issuecomment-1385451139)).

Aligning the PR's CI with the master's configuration to ensure this won't happen again.